lte/gateway/python/magma/pipelined/rpc_servicer.py

```diff
@@ -472,9 +474,100 @@ def UpdateUEState(self, request, context):
             return UESessionContextResponse(operation_type=request.ue_session_state.ue_config_state,
                                             cause_info=CauseIE(cause_ie=CauseIE.REQUEST_REJECTED_NO_REASON))
 
+
     def _setup_pg_tunnel_update(self, request: UESessionSet, fut: 'Future(UESessionContextResponse)'):
-        res = self._classifier_app.process_mme_tunnel_request(request)
-        fut.set_result(res)
+        cause_ie = CauseIE.REQUEST_ACCEPTED
```

Comment:
I actually really preffer the
```
res = self._classifier_app.process_mme_tunnel_request(request)
fut.set_result(res)
```
approach.

The rpc_servier.py should only check if the service is enabled/log grpc payload(`self._log_grpc_payload(request)`). The actual logic is controller speicifc and as such should reside in classifier.py

Should we also add a check to verify the service is running? 
```
        if not self._service_manager.is_app_enabled(
                CLASSIFIER.APP_NAME):
            context.set_code(grpc.StatusCode.UNAVAILABLE)
            context.set_details('Service not enabled!')
            return None
```

Reply:
Changes done as requested.

Reply:
def UpdateUEState(self, request, context):

        self._log_grpc_payload(request)
        if not self._service_manager.is_app_enabled(
              Classifier.APP_NAME):
            context.set_code(grpc.StatusCode.UNAVAILABLE)
            context.set_details('Service not enabled!')
            return None
This check is present in GRPC service "UpdateUEState.